### ISSUE TITLE
[Bug]: incompatiblity with fresco@3.3.0

### ISSUE THREAD

**Issue by efstathiosntonas** [rnmapbox/maps#3650]
### Mapbox Implementation

Mapbox

### Mapbox Version

11.6.0

### React Native Version

0.75.4

### Platform

Android

### `@rnmapbox/maps` version

10.1.31

### Standalone component to reproduce

Fresco@3.3.0 will probably land on rn@0.76 according to this PR: https://github.com/facebook/react-native/pull/46864

edit `node_modules/react-native/gradle/libs.versions.toml` and set `fresco = "3.3.0"`

build fails:

```
> Task :rnmapbox_maps:compileDebugKotlin
e: file:///Users/stathis/IdeaProjects/APP_NAME_mobile/node_modules/@rnmapbox/maps/android/src/main/java/com/rnmapbox/rnmbx/components/mapview/RNMBXMapView.kt:18:27 Unresolved reference: ViewTreeLifecycleOwner
e: file:///Users/stathis/IdeaProjects/APP_NAME_mobile/node_modules/@rnmapbox/maps/android/src/main/java/com/rnmapbox/rnmbx/components/mapview/RNMBXMapView.kt:119:30 Object is not abstract and does not implement abstract member public abstract val lifecycle: Lifecycle defined in com.rnmapbox.rnmbx.components.mapview.RNMBXLifeCycleOwner
e: file:///Users/stathis/IdeaProjects/APP_NAME_mobile/node_modules/@rnmapbox/maps/android/src/main/java/com/rnmapbox/rnmbx/components/mapview/RNMBXMapView.kt:134:17 'getLifecycle' overrides nothing
e: file:///Users/stathis/IdeaProjects/APP_NAME_mobile/node_modules/@rnmapbox/maps/android/src/main/java/com/rnmapbox/rnmbx/components/mapview/RNMBXMapView.kt:138:13 Unresolved reference: ViewTreeLifecycleOwner
e: file:///Users/stathis/IdeaProjects/APP_NAME_mobile/node_modules/@rnmapbox/maps/android/src/main/java/com/rnmapbox/rnmbx/components/styles/RNMBXStyleImport.kt:9:37 Unresolved reference: style
e: file:///Users/stathis/IdeaProjects/APP_NAME_mobile/node_modules/@rnmapbox/maps/android/src/main/java/com/rnmapbox/rnmbx/components/styles/layers/RNMBXLayer.kt:21:37 Unresolved reference: layer

> Task :rnmapbox_maps:compileDebugKotlin FAILED
```

downgrading fresco@3.2.0 builds fine.

**Comment by efstathiosntonas** [comment]
Hey guys, not a big deal for now but we might face this in the future, this is just a "proactive" bug report.

**Comment by efstathiosntonas** [comment]
@mfazekas this should not be closed, rn@0.76 is coming soon.
